From: Georg Lehmann <dadschoorse@gmail.com>
Subject: [PATCH v3 2/2] winevulkan: Enable VK_EXT_pipeline_creation_feedback.
Message-Id: <20220308134051.8268-2-dadschoorse@gmail.com>
Date: Tue,  8 Mar 2022 14:40:51 +0100
In-Reply-To: <20220308134051.8268-1-dadschoorse@gmail.com>
References: <20220308134051.8268-1-dadschoorse@gmail.com>

We did previously not support this extension because it runs into 32bit
alignment issues but those are now fixed.

Signed-off-by: Georg Lehmann <dadschoorse@gmail.com>
---
 dlls/winevulkan/make_vulkan | 1 -
 1 file changed, 1 deletion(-)

diff --git a/dlls/winevulkan/make_vulkan b/dlls/winevulkan/make_vulkan
index 2f3d647ef9b..ca5cf94af55 100755
--- a/dlls/winevulkan/make_vulkan
+++ b/dlls/winevulkan/make_vulkan
@@ -96,7 +96,6 @@ UNSUPPORTED_EXTENSIONS = [
     "VK_AMD_display_native_hdr",
     "VK_EXT_full_screen_exclusive",
     "VK_EXT_hdr_metadata", # Needs WSI work.
-    "VK_EXT_pipeline_creation_feedback",
     "VK_GOOGLE_display_timing",
     "VK_KHR_external_fence_win32",
     "VK_KHR_external_semaphore_win32",

-- 
2.35.1

